hw/ip/prim_generic/prim_generic_rst_sync.core

```diff
@@ -3,15 +3,17 @@ CAPI=2:
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-name: "lowrisc:prim:rst_sync"
-description: "Primitive Reset synchronizer"
+name: "lowrisc:prim_generic:rst_sync"
+description: "Generic implementation of a reset synchronizer"
+virtual:
+  - lowrisc:prim:rst_sync
+
 filesets:
   files_rtl:
     depend:
       - lowrisc:prim:clock_mux2
       - lowrisc:prim:flop_2sync
       - lowrisc:prim:mubi
-      - lowrisc:prim:cdc_rand_delay
```

Comment:
Is this not required anymore?

Reply:
No. The only core that requires `cdc_rand_delay ` is `flop_2sync`, and that already has that on a dependency itself.